setup.py

```diff
@@ -7,46 +7,45 @@
 import re
 from setuptools import setup
 
-with open('flask_meld/__init__.py', 'r') as f:
-    version = re.search(r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]',
-                        f.read(), re.MULTILINE).group(1)
+with open("flask_meld/__init__.py", "r") as f:
+    version = re.search(
+        r'^__version__\s*=\s*[\'"]([^\'"]*)[\'"]', f.read(), re.MULTILINE
+    ).group(1)
 
 HERE = pathlib.Path(__file__).parent
 README = (HERE / "README.md").read_text()
 
 setup(
-    name='Flask-Meld',
+    name="Flask-Meld",
     version=version,
-    url='http://github.com/mikeabrahamsen/Flask-Meld/',
-    license='MIT',
-    author='Michael Abrahamsen',
-    author_email='mail@michaelabrahamsen.com',
-    description='Meld is a framework for Flask that allows you to create dynamic user interfaces using Python and the Jinja2 templating engine.',
+    url="http://github.com/mikeabrahamsen/Flask-Meld/",
+    license="MIT",
+    author="Michael Abrahamsen",
+    author_email="mail@michaelabrahamsen.com",
+    description="Meld is a framework for Flask that allows you to create dynamic user interfaces using Python and the Jinja2 templating engine.",
     long_description=README,
     long_description_content_type="text/markdown",
-    packages=['flask_meld'],
+    packages=["flask_meld"],
     zip_safe=False,
     include_package_data=True,
-    platforms='any',
+    platforms="any",
     install_requires=[
-        'Flask',
-        'beautifulsoup4',
-        'orjson',
-        'flask-socketio',
-        'gevent-websocket'
+        "Flask",
```

Comment:
these don't need to be pinned for now right?

Reply:
Thank you for pointing this out. An issue was opened on the example repo because of this: https://github.com/mikeabrahamsen/Flask-Meld-Example/issues/1

Updated: https://github.com/mikeabrahamsen/Flask-Meld/commit/96cbc12d460a616be553b9c62b176fd0282e1f1c

Reply:
awesome!